styles/components/modal/_core.scss

```diff
@@ -0,0 +1,82 @@
+// Modal structure, taken from ember-modal-dialog/ember-modal-structure
+.ember-modal-overlay {
+  height: 100vh;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  z-index: 50;
+}
+
+.ember-modal-dialog {
+  z-index: 50;
+  position: fixed;
+
+  &.ember-modal-dialog-in-place {
+    position: static;
+  }
+}
+
+// Base modal
+.ember-modal-dialog {
+  border-radius: $adde-modal-border-radius;
+  background-color: $adde-modal-background;
+  width: $adde-modal-width;
+  box-shadow: $adde-modal-box-shadow;
+
+  &.size-sm {
+    width: $adde-modal-width-sm;
+  }
+
+  &.size-lg {
+    width: $adde-modal-width-lg;
+  }
+}
+
+// Overlay
+.ember-modal-overlay.translucent {
+  background-color: $adde-modal-overlay-background;
+}
+
+.adde-modal {
+  font-family: $adde-modal-font-family;
+  @include font-settings();
+}
+
+// Body
+.adde-modal-body {
+  padding: $adde-modal-body-padding;
+}
+
+// Header
+.adde-modal-header {
+  background-color: $adde-modal-header-background;
+  color: $adde-modal-header-text-color;
+  
+  h4.adde-modal-title {
+    margin: 0;
+    padding: $adde-modal-header-padding;
+    height: $adde-modal-header-text-height;
+    font-size: $adde-modal-header-font-size;
+    font-weight: $adde-modal-header-font-weight;
+  }
+
+  button.adde-modal-close {
+    float: right;
+    border: 0;
+    height: $adde-modal-close-button-size;
+    width: $adde-modal-close-button-size;
+    color: $adde-modal-header-text-color;
+    border-radius: $adde-modal-close-button-border-radius;
+
+    &:hover {
+      background-color: $adde-modal-hover-close-background;
+    }
+  }
+}
+
+// Footer
+.adde-modal-footer {
+  padding: $adde-modal-footer-padding;
+  border-top: 1px solid $border-item;
```

Comment:
I'd use the same pattern as the rest of the file and create a new modal var to use here instead of directly using a core var.
`$adde-modal-footer-border: 1px solid $border-item;`